src/test/java/bio/terra/service/search/SearchTest.java

```diff
@@ -0,0 +1,105 @@
+package bio.terra.service.search;
+
+import bio.terra.common.category.Unit;
+import bio.terra.model.SearchIndexRequest;
+import bio.terra.service.resourcemanagement.google.GoogleProjectResource;
+import bio.terra.service.snapshot.Snapshot;
+import bio.terra.service.snapshot.SnapshotTable;
+import bio.terra.service.tabulardata.google.BigQueryPdao;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.mockito.Mockito.when;
+
+@Category(Unit.class)
+public class SearchTest {
```

Comment:
I'm not sure what the current conventions are, but I would expect a test class's name to reflect the class it's testing. So I would expect this to be called `SearchServiceTest`.

Reply:
👍